src/display/editor/freetext.js

```diff
@@ -355,13 +353,6 @@ class FreeTextEditor extends AnnotationEditor {
     }
 
     super.commit();
-    if (!this.#hasAlreadyBeenCommitted) {
-      // This editor has something and it's the first time
-      // it's commited so we can add it in the undo/redo stack.
-      this.#hasAlreadyBeenCommitted = true;
-      this.parent.addUndoableEditor(this);
```

Comment:
Based on a quick search, it appears that this was the *only* call-site using that method; hence please remove the following code since it's now dead:
https://github.com/mozilla/pdf.js/blob/22d350cae6df4caa13f3781074f978ae47f4f956/src/display/editor/annotation_editor_layer.js#L348-L361

Reply:
Also, the following method appears equally unused now:
https://github.com/mozilla/pdf.js/blob/22d350cae6df4caa13f3781074f978ae47f4f956/src/display/editor/annotation_editor_layer.js#L333-L346